fastapi_keycloak_middleware/schemas/keycloak_configuration.py

```diff
@@ -112,3 +115,8 @@ class KeycloakConfiguration(BaseModel):  # pylint: disable=too-few-public-method
         description="The claim that will be checked. If present and if it evaluates to"
         " true, the device authentication will be applied for the request.",
     )
+    verify: Union[bool, str] = Field(
+        False,
```

Comment:
Should default to True, not to false. Would change default behaviour otherwise.

Reply:
@waza-ari cool, I will make that change